src/kedro_profile/hook.py

```diff
@@ -21,9 +21,12 @@
 class ProfileHook:
     env: str | list[str] = "local"
     save_file: bool = False
-    save_path: str | Path = "kedro_pipeline_profile.csv"
-    node_profile_path: str | Path = "node_profile.csv"
-    dataset_profile_path: str | Path = "dataset_profile.csv"
+    node_profile_path: str | Path = (
+        f"data/08_reporting/profiling/node_profile_{time.strftime('%Y%m%d_%H%M%S')}.csv"
```

Comment:
I'd rather not have this path hardcoded in the settings - if anything this could be pass as an argument in the `settings.py`.

```python
HOOKS = (ProfileHook(node_profile_path=...),)
```